Sources/CLMDB/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_library(CLMDB STATIC
```

Comment:
Why explicitly static? Why not drop `STATIC` and let the user control the library type?